sixtyfps_runtime/corelib/item_tree.rs

```diff
@@ -93,6 +93,23 @@ pub enum ItemTreeNode<T> {
     },
 }
 
+impl<T> ItemTreeNode<T> {
+    pub fn parent_index(&self) -> usize {
+        match self {
+            ItemTreeNode::Item { parent_index, .. } => *parent_index as usize,
+            ItemTreeNode::DynamicTree { parent_index, .. } => *parent_index as usize,
+        }
+    }
+    pub fn children_index(&self) -> Option<usize> {
+        match self {
+            ItemTreeNode::Item { item: _, children_count: _, children_index, parent_index: _ } => {
+                Some(*children_index as usize)
+            }
+            ItemTreeNode::DynamicTree { .. } => None,
+        }
+    }
```

Comment:
I don't think we need this one anymore. I'm in favour of removing it. What do you think?